### PR TITLE
Add best trial id to `info` command

### PR DESCRIPTION
**Pull request by bouthilx**

### CODE REVIEW

**Comment by bouthilx** [comment]
[Add best trial id to `info` command](https://trello.com/c/EPwjMxDt/257-add-best-trial-id-to-info-command)

File: src/orion/core/cli/info.py [hunk 1]
```diff
@@ -383,8 +385,9 @@ def format_stats(experiment):
     stats_string = STATS_TEMPLATE.format(
         title=format_title("Stats"),
         stats=stats,
-        best_params=format_dict(best_params, depth=1, width=2))
+        best_params=format_dict(best_params, depth=2, width=2))
 
+    print(format_dict(best_params, depth=2, width=2))
```
Comment (mnoukhov) [inline src/orion/core/cli/info.py]: is this print supposed to be here?

Comment (bouthilx) [reply src/orion/core/cli/info.py]: Indeed, thanks!

**Comment by mnoukhov** [comment]
LGTM

**Pull request merged**